ipfs-api/src/client.rs

```diff
@@ -142,10 +160,41 @@ impl IpfsClient {
         })
     }
 
+    #[cfg(feature = "actix")]
+    fn build_base_request<Req>(
+        &self,
+        req: &Req,
+        form: Option<multipart::Form<'static>>,
+    ) -> Result<Request, Error>
+    where
+        Req: ApiRequest + Serialize,
+    {
+        let url = format!(
+            "{}{}?{}",
+            self.base,
+            Req::PATH,
+            ::serde_urlencoded::to_string(req)?
+        );
+        if let Some(form) = form {
+            Request::build()
+                .method(Req::METHOD.clone())
+                .uri(url)
+                .content_type(form.content_type())
+                .streaming(multipart::Body::from(form))
+                .map_err(From::from)
+        } else {
+            Request::build()
+                .method(Req::METHOD.clone())
+                .uri(url)
+                .finish()
+                .map_err(From::from)
+        }
+    }
```

Comment:
Would it be possible to avoid redeclaring the method stub for the different backends? It seems like adding a type alias for `Request vs. Request<hyper::Body>` and `Response vs. Response<hyper::Body>` would let you do this. I want to avoid repeating code if possible.

Reply:
Good point, completely missed using type aliases to do this. I'll revise it.